third-party/eazy/v1/openapi.yaml

```
@@ -1006,19 +1006,19 @@ paths:
             example: 'contactJid@company.com'
         - in: body
           name: name
-          description: Name of the contact
+          description: The name of the contact
           schema:
             type: string
           example: 'John Doe'
         - in: body
           name: reference
-          description: Reference of the contact
+          description: A reference to the contact
           schema:
             type: string
           example: 'Customer #545'
         - in: body
           name: remarks
-          description: Comment for the contact
+          description: A comment on the contact
```

```suggestion
          description: A comment about the contact
```
Comment on is possible (if we see contact as "location") , but the about constructs contact as the "topic". Here I suggest using about to stress that this is comment about the contact :)